sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -0,0 +1,55 @@
+import multiprocessing
+import pytest
+from unittest.mock import patch, Mock
+from kubeflow.katib import models
+from kubeflow.katib.api.katib_client import KatibClient
+
+@pytest.fixture
+def katib_client():
+    return KatibClient()
+
+@pytest.fixture
+def valid_experiment():
+    return models.V1beta1Experiment(
+        metadata=models.V1ObjectMeta(name="test-experiment"),
+        spec=models.V1beta1ExperimentSpec()
+    )
+
+class TestCreateExperiment:
+    @patch('kubeflow.katib.api.katib_client.client.CustomObjectsApi.create_namespaced_custom_object')
+    def test_create_experiment_success(self, mock_create_namespaced_custom_object, katib_client, valid_experiment):
+        mock_create_namespaced_custom_object.return_value = {"metadata": {"name": "test-experiment"}}
+        experiment = katib_client.create_experiment(valid_experiment)
+        assert experiment == "Experiment default/test-experiment has been created"
+
+    @patch('kubeflow.katib.api.katib_client.client.CustomObjectsApi.create_namespaced_custom_object')
+    def test_create_experiment_timeout_error(self, mock_create_namespaced_custom_object, katib_client, valid_experiment):
+        mock_create_namespaced_custom_object.side_effect = multiprocessing.TimeoutError()
+        with pytest.raises(TimeoutError):
+            katib_client.create_experiment(valid_experiment, namespace="timeout")
+
+    @patch('kubeflow.katib.api.katib_client.client.CustomObjectsApi.create_namespaced_custom_object')
+    def test_create_experiment_runtime_error(self, mock_create_namespaced_custom_object, katib_client, valid_experiment):
+        mock_create_namespaced_custom_object.side_effect = RuntimeError()
+        with pytest.raises(RuntimeError):
+            katib_client.create_experiment(valid_experiment, namespace="runtime")
+
+class TestGetExperiment:
+    @patch('kubeflow.katib.api.katib_client.client.CustomObjectsApi.get_namespaced_custom_object')
+    def test_get_experiment_success(self, mock_get_namespaced_custom_object, katib_client, valid_experiment):
+        mock_response = {"metadata": {"name": "test-experiment"}, "spec": {}, "status": {}}
+        mock_get_namespaced_custom_object.return_value = mock_response
+        experiment = katib_client.get_experiment("test-experiment")
+        assert experiment == valid_experiment
+
```

Comment:
`test_get_experiment_success` is failing currently, can you help me fix this?

```
================================================================ short test summary info ================================================================
FAILED sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py::TestGetExperiment::test_get_experiment_success - AssertionError: assert None == {'api_version': None,\n 'kind': None,\n 'metadata': {'annotations': None,\n              'cluster_name': None,\n     ...
======================================================= 1 failed, 5 passed, 100 warnings in 0.28s =======================================================
make: *** [pytest] Error 1
```

Reply:
As I can see, `get_experiment` returns null object, did you mock the call correctly ?
Should you add patches on the KatibClient, similar as in TrainingClient: https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client_test.py#L181-L195 ?